src/Bridges/intervalbridge.jl

```diff
@@ -48,6 +48,26 @@ function MOI.get(model::MOI.ModelLike, a::MOI.ConstraintDual, c::SplitIntervalBr
     return lower_dual > -upper_dual ? lower_dual : upper_dual
 end
 
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintBasisStatus, c::SplitIntervalBridge)
+    lower_stat = MOI.get(model, MOI.ConstraintBasisStatus(), c.lower)
+    upper_stat = MOI.get(model, MOI.ConstraintBasisStatus(), c.upper)
+    if lower_stat == MOI.NONBASIC_AT_LOWER || upper_stat == MOI.NONBASIC_AT_UPPER
+        Compat.@warn("Single sided constraints should not have basis status:" *
+        " NONBASIC_AT_LOWER or NONBASIC_AT_UPPER, instead use NONBASIC")
+    end
+    if lower_stat == MOI.NONBASIC
+        return MOI.NONBASIC_AT_LOWER
+    end
+    if upper_stat == MOI.NONBASIC
+        return MOI.NONBASIC_AT_UPPER
+    end
+    if lower_stat != upper_stat
+        Compat.@warn("Basis status of lower and upper constraint are inconsistent, " *
+                    "both should be basic or super basic.")
```

Comment:
Missing space in indent. Print the values of `lower_stat` and `upper_stat` in message